app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,167 @@
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+@import 'bootstrap';
+@import "font-awesome";
+
+$blue-color: #3778c2;
+$gray-color: #f5f6f8;
+$green-color: #5fb523;
+$black-color: #434b54;
+
+body {
+  width: 480px;
+  margin: 0 auto;
+  background-color: $gray-color;
+
+  .container {
+    background-color: $gray-color;
+    color: $black-color;
+
+    .loginButtons {
+      display: flex;
+      justify-content: center;
+      padding: 15px;
+
+      * {
+        margin: 0 10px;
+      }
+    }
+  }
+
+  .text-justify {
+    text-align: justify;
+  }
+
+  .wb {
+    background-color: white;
+    margin: 20px 0;
+    border-radius: 5px;
+    box-shadow: 0 3px 10px rgb(209, 208, 208);
+
+    h2,
+    h4 {
+      text-align: center;
+      padding: 10px 0;
+    }
+
+    .transaction,
+    .date {
+      display: flex;
+      justify-content: space-between;
+      padding: 10px;
+    }
+
+    .transaction {
+      border-bottom: 1px solid rgb(209, 208, 208);
+
+      .t-name {
+        font-weight: 700;
+        color: gray;
+      }
+
+      .t-amount {
+        font-weight: bold;
+      }
+    }
+
+    .date {
+      color: rgb(176, 176, 176);
+      font-size: 13px;
+      font-weight: 400;
+    }
+  }
+
+  .navmenu {
+    background-color: $blue-color;
+    color: white;
+
+    .logoMenu {
+      height: 50px;
+      width: auto;
+    }
+
+    .navitem {
+      flex-grow: 1;
+      flex-basis: 0;
+      padding: 10px;
+      display: flex;
+      align-items: center;
+    }
+
+    .al-c {
+      justify-content: center;
+    }
+
+    .al-l {
+      justify-content: flex-start;
+    }
+
+    .al-r {
+      justify-content: flex-end;
+    }
+  }
+
+  .col-md-4 {
+    margin: 10px 0;
+  }
+
+  .tag-item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 8px;
+    min-height: 150px;
+    box-shadow: 0 3px 10px rgb(209, 208, 208);
+
+    a {
+      text-decoration: none;
+    }
+
+    a:hover {
+      color: $green-color;
+    }
+
+    p {
+      padding: 0;
+      margin: 0;
+    }
+
+    .tag-icon-name {
+      display: flex;
+      flex-direction: column;
+    }
+  }
+
+  .g-button {
+    background-color: $green-color;
+    width: 100%;
+    height: 50px;
+    border-radius: 0;
+    border: none;
+    margin: 20px 0;
+  }
+
+  .form-group {
+    padding: 15px 0;
+  }
+}
+
+p.notice,
+p.alert {
+  margin: 0;
+  padding: 0;
```

Comment:
- [ ] As per the project requirement, it is important to ensure that the design of your project aligns with the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). This includes paying attention to aspects such as colors, icons, typography (font face, size, and weight), and layout (composition and space between elements). However, As you can see below, your category page and your transaction page do not meet the required design standards so make sure you adjust the style of your project design and make it responsive to match the design guidelines. :blush: 

  ### Your Design
    
  ![image](https://github.com/zamcham/PocketCanvas/assets/73854999/70606339-800b-4525-88e3-3506e9f3fe18)

![image](https://github.com/zamcham/PocketCanvas/assets/73854999/559a955a-30bf-4230-a40e-56a9ef522ef6)

    ### Expected design
  
  ![image](https://github.com/zamcham/PocketCanvas/assets/73854999/741e47d0-0213-4e99-8131-34259deb67af)